docs/examples/UserGuide/iceloss_ex.jl

```diff
@@ -0,0 +1,84 @@
+using Tyler
+using GLMakie
+using Arrow
+using DataFrames
+using TileProviders
+using Extents
+using ColorSchemes
+using Colors
+using Dates
+
+url = joinpath("https://raw.githubusercontent.com/MakieOrg/Tyler.jl/master/docs/src/assets/data/iceloss_subset.arrow")
+
+## parameter for scaling figure size
+scale = 1;
+
+## load ice loss data [courtesy of Chad Greene @ JPL]
+df = DataFrame(Arrow.Table(url));
+
+## select map provider
+provider = TileProviders.Esri(:WorldImagery);
+
+## Greenland extent
+extent = Extent(X = (-54., -48.), Y = (68.8, 72.5));
+        
+## extract data
+cnt = [length(foo) for foo in df.X];
+X =  reduce(vcat,df.X);
+Y =  reduce(vcat,df.Y);
+Z = [repeat([i],c) for (i, c) = enumerate(cnt)];
+Z = reduce(vcat,Z);
+
+## make color map
+cmap = ColorSchemes.thermal.colors;
+nc = length(cmap);
+alpha = zeros(nc);
+cmap = RGBA.(cmap, alpha);
+cmap0 = Observable(cmap);
+
+## add frame
+frame = Rect2f(extent.X[1], extent.Y[1], extent.X[2] - extent.X[1], extent.Y[2] - extent.Y[1]);
+
+## show map
+m = Tyler.Map(frame; provider, figure=Figure(resolution=(1912 * scale, 2284 * scale)));
+
+## create initial scatter plot
+n = nrow(df);
+scatter!(m.axis, X, Y; color = Z, colormap = cmap0, colorrange = [0, n], markersize = 10);
+
+# add color bar
+a,b = extrema(df.Date);
+a = year(a);
+b = year(b);
+Colorbar(m.figure[1,2]; colormap = cmap0, colorrange = [a,b], 
+    ticklabelsize = 50 * scale, width = 100 * scale);
+
+## hide ticks, grid and lables
+hidedecorations!(m.axis);
+
+## hide frames
+hidespines!(m.axis);
+
+## loop to create animation
+for k = 1:15
+
+    # reset apha
+    i = 1;
+    cmap = ColorSchemes.thermal.colors;
```

Comment:
```suggestion
    cmap = RGBf.(Makie.to_colormap(:thermal))
```
avoids the explicit colorschemes dependency, and is guaranteed to return a vector of RGBAf which is then converted to RGBf.